### DISCUSSION TITLE
running out of disk space

### DISCUSSION THREAD

**Discussion by z3d3m0n** [postalserver/postal#2280] [Help with using Postal]
Hello commmunity,

while using postal with some small transactional tasks, it was working fine.

When i tried to load a bit more traffic with a 200k send out, it took almost 12 hours to send with 150 emails per minute speed and
the message queue filed up and used up all the space.
[142154 queued messages]
[306 GB used]

is there a way how to throttle queue size to avoid server crushing when all the space is used up?

thx

**Comment by willpower232** [comment]
The queue has to exist somewhere so I don't think there is too much you could do other than get more storage capacity for your server, the messages will also hang around in Postal for a bit so even once the queue has processed, the storage will still be used.
